integrated_channels/cornerstone/exporters/content_metadata.py

```diff
@@ -93,6 +93,9 @@ def transform_is_active(self, content_metadata_item):
         Return the transformed version of the course is_active status by traversing course runs and setting IsActive to
         True if any of the course runs have availability value set to `Current`, `Starting Soon` or `Upcoming`.
         """
+        # lets not overwrite something we've already tried to set INACTIVE
+        if content_metadata_item.get('IsActive') is not None and content_metadata_item.get('IsActive') is False:
```

Comment:
this is just for safety, might not be 100% needed. eventually we'll want per-action (create, update, delete) transformations which would allow us to drop this kind of logic.